README.md

```diff
@@ -65,6 +65,13 @@ $ cd fedora
 $ packer build -var 'mirror=http://mirror.utexas.edu/fedora/linux' fedora-31-x86_64.json
 ```
 
+To build a Windows 10 Enterprise Gen 2 box for only the Hyper-V provider
+
+```
+$ cd windows
+$ packer build windows-10gen2.json
+```
+
 If the build is successful, ready to import box files will be in the `builds` directory at the root of the repository.
```

Comment:
I'm confused about what happens in this sentence. What does "ready to import box files" mean?

Reply:
I'll fix this in the readme. It should read like "If your build is successful, your box files will be in a builds directory..."